src/org/python/core/RegistryKey.java

```diff
@@ -0,0 +1,297 @@
+     * will be shown on stderr. 
+     *
+     */
+    public static final String PYTHON_OPTIONS_SHOW_PYTHON_PROXY_EXCEPTIONS = 
+        "python.options.showPythonProxyExceptions";
+
+    /**
+     * 
+     * {@code python.options.proxyDebugDirectory} is the directory where 
+     * dynamically generated classes are written. 
+     * Nothing is ever read from here, it is only for debugging purposes.
+     *
+     */
+    public static final String PYTHON_OPTIONS_PROXY_DEBUG_DIRECTORY = 
+        "python.options.proxyDebugDirectory";
+
+    /**
+     *
+     * {@code python.options.Qnew} controls whether true division is enabled 
+     * for the / operator. 
+     *
+     * See PEP 238.  Boolean.
+     *
+     * Equivalent to -Qnew on the command line.
+     */
+    public static final String PYTHON_OPTIONS_Q_NEW = "python.options.Qnew";
+
+    /**
+     *
+     * {@code python.os} defines the string used to report the underlying 
+     * operating system. Used as prefix when resolving which operating system, 
+     * impacting * some OS-specific behaviour.
+     */
+    public static final String PYTHON_OS = "python.os";
+
+    /**
+     *
+     * {@code python.packages.fakepath} defines a sequence of directories 
+     * and JARs * that are to be sources of Python packages.
+     */
+    public static final String PYTHON_PACKAGES_FAKEPATH = 
+        "python.packages.fakepath";
+
+    /**
+     *
+     * {@code python.packages.paths} defines a sequence of property names. Each 
+     * property is a path string. The default setting causes directories and 
+     * JARs on the classpath and in the JRE (before Java 9) to be sources of 
+     * Python packages.
+     */
+    public static final String PYTHON_PACKAGES_PATHS = "python.packages.paths";
+
+   /**
+     * {@code python.packages.directories} defines a sequence of 
+     * property names. Each property name is a path string, 
+     * in which the elements are directories. 
+     * Each directory contains JAR/ZIP files that are to be a source of 
+     * Python packages. By default, these directories are those where the JVM 
+     * stores its optional packages as JARs (a mechanism withdrawn in Java 9).
+     */
+    public static final String PYTHON_PACKAGES_DIRECTORIES = 
+        "python.packages.directories";
+
+    /**
+     *
+     * {@code python.path} is the search path for Python modules, equivalent
+     * to CPython's {@code PYTHONPATH} environment variable.
+     */
+    public static final String PYTHON_PATH = "python.path";
+
+    /**
+     * 
+     * If this {@code python.security.respectJavaAccessibility} is false, 
+     * and you are using a Java version before Java 9, then Jython can 
+     * access non-public fields, methods, and constructors. 
+     * Normally, Jython can only provide access to public members of classes. 
+     * This may be deprecated in the future due to Java changes 
+     * to accessibility from version 9. Boolean.
+     */
```

Comment:
The JVM options ``--permit-illegal-access`` and ``--illegal-access=warn`` could be hinted in this context.

Reply:
Added.